internal/manager/config/config.go

```diff
@@ -129,6 +129,10 @@ const (
 	// rather than use the embedded UI.
 	CustomUILocation = "custom_ui_location"
 
+	// Gallery Cover Regex
+	GalleryCoverRegex        = "gallery_cover_regex"
```

Comment:
since this will be manualy edited in  the config for now, add some info to the in-app manual in the relevant section? <https://github.com/stashapp/stash/blob/develop/ui/v2.5/src/docs/en/Manual/Configuration.md#advanced-configuration-options>